web/cobrands/sass/_layout.scss

```diff
@@ -861,10 +861,6 @@ textarea.form-error {
   }
 }
 
-#front-howto {
```

Comment:
This isn't the right bit to try and remove, which is about 30 lines further down - but we don't want to remove this base style because eg Oxfordshire are also using it.